packages/labs/scoped-registry-mixin/package.json

```diff
@@ -25,6 +25,7 @@
   "files": [
     "/src/",
     "!/src/test/",
+    "/development/scoped-registry-mixin.{d.ts,d.ts.map,js,js.map}",
```

Comment:
```suggestion
    "/development/",
    "!/development/test/",
```

So that if any new source files are added they'll get picked up too, and to not publish test sources.

Reply:
the reason I didn't do it like so was that there is also a test folder in the development folder (see screenshot)

<img width="300" alt="Screen Shot 2021-04-06 at 6 25 54 am" src="https://user-images.githubusercontent.com/21725/113622970-0738dc00-96a1-11eb-9d6a-2f52bbbba359.png">

Let me know if you still want me to change it ?